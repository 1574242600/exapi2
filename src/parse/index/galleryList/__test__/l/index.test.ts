import fs from "fs";
import cheerio from "cheerio";
import parseGalleryItem from "../../l/index";

// eslint-disable-next-line @typescript-eslint/ban-ts-comment
///@ts-ignore
const $ = cheerio.load(fs.readFileSync(global.__DATA_PATH__.html.index.l));

const tr = $("table[class='itg gltc']>tbody>tr").slice(1).eq(0);

test("解析画廊条目信息Html 类型L", () => {
    const info = parseGalleryItem(tr);
    expect(info).toMatchSnapshot();
});