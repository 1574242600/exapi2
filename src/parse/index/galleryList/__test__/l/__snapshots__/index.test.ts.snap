// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`解析画廊条目信息Html 类型L 1`] = `
Object {
  "cover": "https://ehgt.org/t/27/87/27875b5d97e37fb41533aa01e090f34d886407d5-897212-982-1200-jpg_250.jpg",
  "haveBt": true,
  "id": "1854284",
  "pages": 42,
  "posted": 1614050100,
  "rating": 4.5,
  "tags": Object {
    "artist": Array [
      "shibekawa",
    ],
    "female": Array [
      "big ass",
      "big breasts",
      "dark skin",
      "milf",
    ],
  },
  "title": "[Artist] Shibekawa",
  "token": "19459d9323",
  "type": "Image Set",
  "uploader": "Godboli",
}
`;
